tools/points-service/main.go

```diff
@@ -838,7 +838,6 @@ func main() {
 
 			go func() {
 				for err := range sub.Err() {
-					ps.SetSubscriptionActive(false)
```

Comment:
Why was this removed?

Reply:
It was incorrectly setting it to false, there's some common errors that can occur while processing events we don't want procssed, and it shouldn't be setting to inactive, since events are still being subscribed to